src/terminal/adapter/InteractDispatch.cpp

```diff
@@ -188,7 +188,7 @@ bool InteractDispatch::IsVtInputEnabled() const
 //   which will end up here. This will update the console's internal tracker if
 //   it's focused or not, as to match the end-terminal's state.
 // - Used to call ConsoleControl(ConsoleSetForeground,...).
-// - Full support for this sequence is tracked in GH#11682.
+// - Full support for this sequence was added in GH#11682.

```

Comment:
We can just kill this line. Otherwise we're gonna litter our code with historical artifacts pointing to GitHub issues that we could relegate to `git blame` :)